semantic-python/src/Language/Python/Core.hs

```diff
@@ -104,6 +135,12 @@ instance Compile Py.IfStatement where
           clause (Left  Py.ElifClause{ condition, consequence }) rest  =
             if' <$> compile condition <*> compile consequence <*> rest
 
+  compileCC Py.IfStatement{ condition, consequence, alternative} cc =
```

Comment:
Can we define `compile` in terms of our new definition of `compileCC`? E.g. `compile p = compileCC p none`?